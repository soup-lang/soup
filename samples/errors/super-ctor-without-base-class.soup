class SomeClass
    ctor(name) super(name)
        0
