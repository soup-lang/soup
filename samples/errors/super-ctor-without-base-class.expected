samples/errors/super-ctor-without-base-class.soup:2
    no super class
